src/Microsoft.Azure.WebJobs.Host/Filters/FunctionExceptionContext.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host.Filters
+{
+    /// <summary>
+    /// Base context class for <see cref="IFunctionExceptionFilter.OnExceptionAsync(FunctionExceptionContext, System.Threading.CancellationToken)"/>.
+    /// </summary>
+    public class FunctionExceptionContext : FunctionFilterContext
```

Comment:
Because exception filters can run before any input arguments are bound, the base class isn't FunctionInvocationContext (i.e. no Arguments dictionary). We might try to conditionally expose the arguments if the exception happened after they were bound. Thoughts?